app/static/styles/app.css

```diff
@@ -16,7 +16,7 @@ nav .mobile.only.row .ui.vertical.menu {
 
 @media -sass-debug-info{filename{}line{font-family:\0000325}}
 .flashes > .ui.message:last-of-type {
-  margin-bottom: 2rem;
```

Comment:
When you make a scss file change, you should reload the webpage so that the changes can be precompiled into the css file (it's annoying I know) - you can check by seeing what changes were made in `app.css` and they should have the changes you made in `app.scss`